### PR TITLE
Chore/update node version

### PR DESCRIPTION
**Pull request by danielcooperxyz**
Update and pin node at 16.13.2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "url": "git+https://github.com/digicatapult/vitalam-demo-client.git"
   },
   "engines": {
-    "node": ">=16.13.2"
+    "node": "16.13.2"
```
Comment (mattdean-digicatapult) [inline package.json:13]: Open question. Should we be enforcing an exact version here? Perhaps we should be saying `16.x`? Also should we add to engines the `npm` version?

Comment (darrylmorton-digicatapult) [reply package.json:13]: I think `>=16.13`, and yes I think it's worth us adding `npm` version likewise.
The `Dockerfile` builds should follow the same convention that we implement within the `package.json` engines `npm` version

Comment (mattdean-digicatapult) [reply package.json:13]: Our discussion on slack @darrylmorton-digicatapult seems to have settled on `16.x.x` for node and then also specifying npm at a similar level so `8.x.x`. Is that what you meant?

Comment (mattdean-digicatapult) [reply package.json:13]: But yes we should make sure any Dockerfiles and workflows deal with appropriate node and npm versions

**Review by mattdean-digicatapult — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "url": "git+https://github.com/digicatapult/vitalam-demo-client.git"
   },
   "engines": {
-    "node": ">=16.13.2"
+    "node": "16.x.x"
```
Comment (mattdean-digicatapult) [inline package.json:13]: Add npm engine here

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**